src/controller/articleController.js

```diff
@@ -0,0 +1,112 @@
+import { prisma } from '../utils/prismaClient.js';
+
+export class ArticleController {
+  //게시글 목록 조회
+  static getArticles = async (req, res) => {
+    const { offset = 0, limit = 10, order, search } = req.query;
+    const orderByOption = {
+      recent: { createdAt: 'desc' },
+      oldest: { createdAt: 'asc' },
+    };
+    const where = search
+      ? { OR: [{ title: { contains: search } }, { content: { contains: search } }] }
+      : {};
+
+    const article = await prisma.article.findMany({
+      where,
+      orderBy: orderByOption[order] || orderByOption['recent'],
+      skip: parseInt(offset),
+      take: parseInt(limit),
+      select: { id: true, title: true, content: true, createdAt: true },
+    });
+    res.status(200).send(article);
+  };
+  //게시글 생성
```

Comment:
💊  마이너하지만 함수의 끝에 마지막 한줄을 추가해주시면 읽기 더 쉬울 것 같습니다 :)


```suggestion
  };
  
  //게시글 생성
```